web/pages/api/signin.js

```diff
@@ -0,0 +1,39 @@
+import nextConnect from 'next-connect'
+import bcrypt from 'bcrypt'
+import { query } from '../../utils/query'
+
+const handler = nextConnect()
+
+    .post(async (req, res) => {
+        const { email, password } = req.body 
+
+        if (!email || !password) res.status(400).json({ error: 'Email or Password cannot be blank' })
+
+        try {
+            const results = await query(`
+              SELECT password FROM users WHERE email = ?
+            `, email)
```

Comment:
Prepared Statements have to pass in their fields in an array, so you'd have to wrap email in an array.

`[email]`